.github/workflows/release.yaml

```diff
@@ -47,6 +47,50 @@ jobs:
 
           script/release
 
+      - name: Upload build artifacts
+        uses: actions/upload-artifact@v2
+        with:
+          name: dist
+          path: ./dist/
+
+  upload-release-assets:
+    name: Upload release assets
+    runs-on: ubuntu-latest
+    needs: release
+    if: github.event_name == 'release'
+    permissions:
+      contents: write
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v2
+      - name: Download build artifacts
+        uses: actions/download-artifact@v2
+      - name: Get build file names
+        id: file-names
+        run: |
+          echo "::set-output name=sdist::$(ls dist/ | grep '.*.tar.gz' | tail -n 1)"
+          echo "::set-output name=wheel::$(ls dist/ | grep '.*.whl' | tail -n 1)"
+      - name: Upload release asset (sdist)
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{ github.event.release.upload_url }}
+          asset_path: ./dist/${{ steps.file-names.outputs.sdist }}
+          asset_name: ${{ steps.file-names.outputs.sdist }}
+          asset_content_type: application/gzip
+      - name: Upload release asset (wheel)
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{ github.event.release.upload_url }}
+          asset_path: ./dist/${{ steps.file-names.outputs.wheel }}
+          asset_name: ${{ steps.file-names.outputs.wheel }}
+          asset_content_type: application/zip
+      - name: Print release url
+        run: echo "Release assets uploaded! View release at ${{ github.event.release.html_url }}"
```

Comment:
```suggestion
```

I'd say this is here for debug purposes, and should not be added to the workflow